Installing graoupoo/core v0.1.10 produces an error

**Describe the bug**
When installing `@grouparoo/core` version 0.1.10, an error occurs and the install process fails

**Stack Trace**
If applicable, add the full stack trace you see along with the error.

```
src/models/Destination.ts:548:16 - error TS2322: Type '{ [in]: unknown[]; }' is not assignable to type 'string | number | boolean | Date | Buffer | WhereAttributeHash | AndOperator | OrOperator | Literal | ... 5 more ... | (string | ... 2 more ... | WhereAttributeHash)[]'.
  Types of property '[Op.in]' are incompatible.
    Type 'unknown[]' is not assignable to type 'string | number | boolean | Date | Buffer | WhereAttributeHash | AndOperator | OrOperator | Literal | ... 6 more ... | (string | ... 2 more ... | WhereAttributeHash)[]'.
      Type 'unknown[]' is not assignable to type '(string | number | Literal)[]'.
        Type 'unknown' is not assignable to type 'string | number | Literal'.
          Property 'val' is missing in type '{}' but required in type 'Literal'.

548       where: { guid: { [Op.in]: recentGroupGuids }, state: "updating" },
                   ~~~~

  ../../../sequelize/types/lib/utils.d.ts:100:10
    100   public val: unknown;
                 ~~~
    'val' is declared here.
```

--- a/core/api/src/models/Destination.ts
+++ b/core/api/src/models/Destination.ts
@@ -325,7 +325,7 @@ export class Destination extends LoggedModel<Destination> {
     return DestinationOps.destinationMappingOptions(this);
   }
 
-  async getRuns(state?: string, limit = 100, offset = 0) {
+  async getRuns(state?: string, limit = 100, offset = 0): Promise<Run[]> {
     const exportRuns = await ExportRun.findAll({
       raw: true,
       attributes: [
@@ -536,13 +536,13 @@ export class Destination extends LoggedModel<Destination> {
   @BeforeDestroy
   static async waitUpdatingGroupsPreviouslySynced(instance: Destination) {
     const recentRuns = await instance.getRuns(undefined, 1, 0);
-    const recentGroupGuids = [
-      ...new Set(
+    const recentGroupGuids = Array.from(
+      new Set(
         recentRuns
           .filter((run) => run.creatorGuid.match(/^grp_/))
           .map((run) => run.creatorGuid)
-      ),
-    ];
+      )
+    );
     const updatingGroups = await Group.findAll({
       where: { guid: { [Op.in]: recentGroupGuids }, state: "updating" },
     });

